### PR TITLE
Update to easyrsa 3.0, remove .git dir

### PR DESCRIPTION
**Pull request by jvassev**

### CODE REVIEW

**Comment by kylemanna** [comment]
Good catch on the release. :+1:  I think now that Easy RSA v3 is out we can remove git-core from the pkg install list and use the tarball.

If you could update your branch (re-base please to keep the history clean) to use the release tarball @ https://github.com/OpenVPN/easy-rsa/archive/v3.0.0.tar.gz and remove the chmod stuff I commented on I'd love to merge it.

File: Dockerfile [hunk 1]
```diff
@@ -9,9 +9,10 @@ RUN apt-get update && \
     apt-get install -y openvpn iptables git-core && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Update checkout to use tags when v3.0 is finally released
-RUN git clone --depth 1 --branch v3.0.0-rc2 https://github.com/OpenVPN/easy-rsa.git /usr/local/share/easy-rsa && \
-    ln -s /usr/local/share/easy-rsa/easyrsa3/easyrsa /usr/local/bin
+RUN git clone --depth 1 --branch v3.0.0 https://github.com/OpenVPN/easy-rsa.git /usr/local/share/easy-rsa && \
+	rm -fr /usr/local/share/easy-rsa/.git && \
```
Comment (kylemanna) [inline Dockerfile]: Good catch.  Should have been cleaned up.

File: Dockerfile [hunk 2]
```diff
@@ -28,4 +29,4 @@ WORKDIR /etc/openvpn
 CMD ["ovpn_run"]
 
 ADD ./bin /usr/local/bin
-RUN chmod a+x /usr/local/bin/*
```
Comment (kylemanna) [inline Dockerfile]: Why get rid of this one line and replace it with 2 different ones?

**Comment by ypid** [comment]
Please also check the signature now that we can use the tarball: https://github.com/OpenVPN/easy-rsa/releases/download/3.0.0/EasyRSA-3.0.0.tgz.sig
We could/should have done this also for the [v3.0.0-rc2](https://github.com/OpenVPN/easy-rsa/releases/tag/v3.0.0-rc2)

**Comment by ypid** [comment]
@jvassev You forgot about checking the sig. See https://github.com/jchaney/owncloud/pull/12

**Comment by kylemanna** [comment]
@tsing Any idea if Easy RSA will ever be in Alpine's main?  I started playing with Alpine the yesterday and like it alot.

**Pull request merged**